### PR TITLE
Batch replay archiving

### PR DESCRIPTION
**Pull request by blinktag**

### CODE REVIEW

File: backends/batch/replays.go [hunk 1]
```diff
@@ -90,6 +92,40 @@ func (b *Batch) listReplays() ([]ReplayOutput, error) {
...
+		if err := json.Unmarshal(res, errResponse); err != nil {
+			return errReplayArchiveFailed
+		}
+
+		for _, e := range errResponse.Errors {
+			err := fmt.Errorf("%s: %s", errReplayArchiveFailed, e.Message)
+			b.Log.Error(err)
+		}
+
+		return err
```
Comment (dselans) [inline backends/batch/replays.go:123]: does this just return `nil` in all cases? This should be returning a `Non-2xx Error`.

File: backends/batch/replays.go [hunk 2]
```diff
@@ -90,6 +92,40 @@ func (b *Batch) listReplays() ([]ReplayOutput, error) {
...
+	return nil
+}
+
+func (b *Batch) archiveReplay() error {
+	res, code, err := b.Delete("/v1/replay/" + b.Opts.Batch.ReplayID)
+	if err != nil {
+		return errors.Wrap(err, errReplayArchiveFailed.Error())
+	}
+
+	if code > 299 {
```
Comment (dselans) [inline backends/batch/replays.go:112]:
```suggestion
	if code > 299 || code < 200 {
```

**Review by dselans — APPROVED** [review]
Two minor things.

File: backends/batch/replays.go (hunk 2 above)
Comment (blinktag) [reply backends/batch/replays.go:112]: Fixed!

File: backends/batch/replays.go (hunk 1 above)
Comment (blinktag) [reply backends/batch/replays.go:123]: Yeah good catch, returning an actual error now

**Pull request merged**